utils/src/main/kotlin/CopyrightStatementsProcessor.kt

```diff
@@ -113,13 +115,13 @@ class CopyrightStatementsProcessor {
 
     data class Result(
         val processedStatements: Map<String, List<String>>,
-        val unprocessedStatements: List<String>
+        val unprocessedStatements: SortedSet<String>
```

Comment:
Just to point out the order-topic again: It seems a bit odd that you drop maintaining the order of `processedStatements` in the previous commit, but introduce maintaining the order of `unprocessedStatements` here.

Reply:
See above.